Source/WebCore/platform/network/ResourceResponseBase.cpp

```diff
@@ -131,13 +132,13 @@ ResourceResponse ResourceResponseBase::fromCrossThreadData(CrossThreadData&& dat
     ResourceResponse response;
 
     response.setURL(data.url);
-    response.setMimeType(AtomString { data.mimeType });
```

Comment:
Why do we not need to create isolated copies for all objects like `data.url` too?

Reply:
AtomString is modifying the given strings, hence the issue. data.url remains an URL.
Next version will hopefully be clearer.